napari_allencell_annotator/view/annotator_view.py

```diff
@@ -80,19 +84,21 @@ class AnnotatorView(QFrame):
     def __init__(
         self,
         model: AnnotatorModel,
-        viewer: Viewer,
+        viewer: IViewer,
         mode: AnnotatorViewMode = AnnotatorViewMode.ADD,
     ):
         super().__init__()
         self._annotator_model = model
         self._annotator_model.image_changed.connect(self._handle_image_changed)
+        self._annotator_model.edit_points_layer_changed.connect(self._handle_point_selection)
+        self._annotator_model.current_annot_changed.connect(self._handle_item_changed)
```

Comment:
this should directly subscribe to the templateitem signal

Reply:
subscribed in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/4c41eedbf3fcdd7fa4c80ee82a2ed87febfe233f